packages/amplify-migration-tests/src/migration-helpers/utils.ts

```diff
@@ -170,3 +171,21 @@ export const pullPushWithLatestCodebaseValidateParameterAndCfnDrift = async (pro
     deleteProjectDir(projRoot2);
   }
 };
+
+/**
+ * Pulls and pushes with force project with latest codebase. Validates parameter and cfn drift.
+ */
+export const pullPushForceWithLatestCodebaseValidateParameterAndCfnDrift = async (projRoot: string, projRoot2: string): Promise<void> => {
+  const appId = getAppId(projRoot);
+  expect(appId).toBeDefined();
+  try {
+    await amplifyPull(projRoot2, { emptyDir: true, appId }, true);
+    assertNoParameterChangesBetweenProjects(projRoot, projRoot2);
+    expect(collectCloudformationDiffBetweenProjects(projRoot, projRoot2)).toMatchSnapshot();
+    await amplifyPushForce(projRoot2, true);
```

Comment:
Added a forcePush function as we need a mechanism to trigger changes on amplifuPush . Changes are only related to build files.

Reply:
please add comment in the code.